packages/core/src/DatumBuilders/DatumBuilder.V1.class.ts

```diff
@@ -114,26 +102,22 @@ export class DatumBuilderV1 implements DatumBuilderAbstract {
     orderAddresses,
     deposit,
     scooperFee,
-  }: IDepositArguments): TDatumResult<TDepositOrder> {
-    const datum: TDepositOrder = {
+  }: IDepositArguments): TDatumResult<V1Types.DepositOrder> {
+    const datum: V1Types.DepositOrder = {
       ident: this.buildPoolIdent(ident),
       orderAddresses: this.buildOrderAddresses(orderAddresses).schema,
       scooperFee,
       DepositPair: {
-        Parent: {
-          Child: {
-            Value: {
-              pair: {
-                a: deposit.CoinAAmount.amount,
-                b: deposit.CoinBAmount.amount,
-              },
-            },
+        Child: {
+          pair: {
```

Comment:
Does this still result in the same level of wrapping? I think we have tests that cover this datum builder, so as long as those are the same then we're good.